### ISSUE TITLE
Connected devices research activity

### ISSUE THREAD

**Issue by ezoehunt** [mozilla/Community-Design#8]
Since Orlando, the Connected Devices design team has been researching the Internet of Things. It's a huge space with lots of possibilities for Mozilla impact! 

Right now we're doing an activity to identify more focused areas of exploration. We thought this might be a fun exercise for other people to do with us - contributors and staff from other areas of Mozilla. So this issue is an experiment in how we might do research/brainstorm activities via Github issues. 

Here's a [link to the instructions][1]. Our team will use the outcomes from this exercise to focus our ongoing brainstorming activities for IoT.

If you have any thoughts or feedback on how we can improve the format of issues like these, please post in the comments below...so we can do better at this in the future.

Thanks!

[1]: https://docs.google.com/document/d/1haxXDs7cPMQCKWhzyI62Lc1FU7Nr-U1aJ8vwFSs5aWI

**Comment by Jobava** [comment]
Hello, I would like to add an important and growing demographic: "security-conscious people".

Internet of Things is practically today "internet of other people's things", and also proprietary and badly secured. Mozilla has a reputation as a trustworthy community which promotes freedom and openness. Mozilla could promote a "**radically open**" IoT concept where people get devices they can trust and provably so, via an open-source/free-software model.

In Germany, Firefox is still the leading browser and Facebook usage is very low compared to other developed nations. Not only that, Facebook usage is **inversely correlated** with education, which means the young and educated are increasingly avoiding it. I believe that's the start of a trend and in the near future we're going to have a real market for products that cater to the privacy-conscious.

Links:
http://www.dw.com/en/educated-germans-avoid-social-media/a-18875970
http://gs.statcounter.com/#browser-DE-monthly-201412-201512
"The coming war on general purpose computation" by Cory Doctorow: https://www.youtube.com/watch?v=gbYXBJOFgeI

**Comment by ezoehunt** [comment]
@Jobava: thanks for adding that. We're not quite at the solution phase yet - we're still exploring the space. Security is definitely on our radar. If you'd like to write up a scenario that focuses on "security-conscious people," that would be great.

**Comment by parag9d** [comment]
I have been going through CES 2016 and IoT was the at the heart of it. I would love to dig deeper into this project and I am keen to contribute to the UX Design for Connected Devices. I am going through the doc and will try to find a combination to explore. Will try to come up with some ideas soon.

**Comment by LucyeoH** [comment]
Hey @ezoehunt this is cool idea but I think I'm missing the design component? Is this a community-design project or just research for IoT? If so, it might be a better fit in the [Participation Leaders Repo here](https://github.com/mozilla/participation-leaders-accountability/).

**Comment by elioqoshi** [comment]
@LucyeoH Can we imigrate this to the other repo and close this or how should we proceed?

**Comment by ezoehunt** [comment]
@LucyeoH I've closed this issue, as our team has what we need to move forward. 

When the Mozilla design teams were introduced to this repo, we talked about using it for design-related tasks like user research and other design-related activities. As Mozilla moves towards a more holistic UCD process, there will be more activities like this, and we'd like to include community designers in them. 

In my mind, this repo is the right place to extend those opportunities to the community. But perhaps there's been further discussion by others that I've missed...
